docs/reference/release-notes/850.md

```diff
@@ -4,11 +4,11 @@ title: "8.5 Release notes"
 description: "Release notes for 8.5, including alphas"
 ---
 
-Release date: 9th of April 2024
+## 8.5 minor
 
-End of maintenance: 14th of October 2025
-
-## Enhancements
+| Release date | End of maintenance | Changelog(s)                                                                                                                                                         |                                                                       |
```

Comment:
The last column should have "Release Blog" as heading, isn't it?

Reply:
Since the link is "Release blog," I thought that looked noisy/redundant. I will leave it for now without a table header.